lib/Components/core.css

```diff
@@ -138,7 +138,7 @@
         --COLOR-FONT: black;
         --COLOR-SCROLL:#ffffff;
         --COLOR-OUTLINE-BTN-BG:#ffffff;
```

Comment:
--COLOR-SCROLL and --COLOR-OUTLINE-BTN-BG have same hex. Reason?

Reply:
okay will check